This is my sixth homework. Header and intro title.

Billiardist94 opened the pull request:
Working with main title. Header and intro.

Commit:
This is my sixth homework. Header and intro title.

romos21 commented on index.html:
```diff
@@ -7,6 +7,77 @@
+                <ul class="header-list">
+                    <li class="header-item"><a href="/" class="header-link">Stays</a></li>
+                    <li class="header-item"><a href="/" class="header-link">Attractions</a></li>
+                    <li class="header-item"><a href="/" class="header-link"><img src="images/moon-icon.svg" alt="night-theme"></a></li>
```
@Billiardist94 I think it's not a link and it's better to remove A tag here

romos21 commented on style.css:
```diff
@@ -10,14 +10,190 @@
+    font-size: 50px;
+    line-height: 59px;
+    color: #FFFFFF;
+    max-width: 542px;
```
@Billiardist94 I remove this property and didn't see the difference

romos21 commented:
@Billiardist94 [
![image](https://user-images.githubusercontent.com/57193802/110241461-f68e2c80-7f61-11eb-8377-37c47707ef65.png)
](url) Maybe you can set any distance near the border container

romos21 commented:
@Billiardist94 I think there are some places where you use too many wrappers like this [
![image](https://user-images.githubusercontent.com/57193802/110241549-7ddba000-7f62-11eb-9466-5f7eff82009f.png)
](url) -padding for one and width for its wrapper - maybe try to set two properties to one wrapper and remove other

Commit:
This is my sixth homework. Header and intro title.
Remove header-wrapper;
Make distance between apps-form and border;
Remove "a" tag in the header;
Make a line.
